### ISSUE TITLE
[v4] | [v2] Lacking Accessibility Support

### ISSUE THREAD

**Issue by llochridge0** [gorhom/react-native-scroll-bottom-sheet#1860]
# Bug

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | 4.6.1 |
| react-native                    | 0.72.14   |
| react-native-reanimated         | 3.3.0   |
| react-native-gesture-handler    | 2.9.0  |

## Steps To Reproduce

Open Xcode -> click on top bar (Xcode) -> Open Developer tools -> Accessibility Inspector -> Choose Simulator or device with app (that is using Bottomsheet) -> Navigate in app to location with bottomsheet. 
1. Inspect contents in bottom sheet. There are no accessibility targets other than the entire bottom sheet. label: Bottom sheet backdrop, hint: Tap to close the bottom sheet and Traits: Button

Describe what you expected to happen:

1. When inspecting a bottomsheet each of the components/content within should retain its own accessibility labels, roles and hints

On the site it states that Gohorm BottomSheet supports accessibility, so I guess that could be subjective. Is what I'm encountering the expected outcome?

**Comment by kkureli** [comment]
Hi, any solution?

**Comment by llochridge0** [comment]
> Hi, any solution?

Sadly I have neither seen nor heard anything on this. Best solution at this point seems to be, find an alternative.

**Comment by kkureli** [comment]
> > Hi, any solution?
> 
> Sadly I have neither seen nor heard anything on this. Best solution at this point seems to be, find an alternative.

Thanks for reply. Do you mean another bottom sheet library as an alternative?

**Comment by llochridge0** [comment]
> > > Hi, any solution?
> > 
> > 
> > Sadly I have neither seen nor heard anything on this. Best solution at this point seems to be, find an alternative.
> 
> Thanks for reply. Do you mean another bottom sheet library as an alternative?

Yes, I will likely have to find an alternate bottom sheet library (or make my own implementation)

**Comment by eirikluka** [comment]
The whole BottomSheet has the property `accessible={true}` as default. If you set `accessible={false}` you should be able to navigate using VoiceOver / Accessibility Inspector. 

I don't know why the default is `true`, but it might be a misunderstanding of what the property does. I've made that mistake myself.

[Here's a PR](https://github.com/gorhom/react-native-bottom-sheet/pull/1788) from April which solves this.

**Comment by idanlevi1** [comment]
By default is `accessible: true`, so in IOS does not move between elements inside so my solution is:
```
<BottomSheetElement
            accessible={Platform.OS === 'android'}
            ...
```

**Comment by zaferatli** [comment]
The maintainer isn’t interested in these kinds of issues. I tried reaching out in many ways, but he never responded or showed any interest. https://github.com/gorhom/react-native-bottom-sheet/pull/2059

**Comment by williamsaar** [comment]
Been diving into this problem also. I can confirm what @zaferatli says - all the issues mentioning accessibility seems to be ignored. Only solution seems to make our own hacky way to make the bottom-sheet accessible...

**Comment by arapocket** [comment]
This still is an issue
